packages/chat-component/src/components/chat-stage.ts

```diff
@@ -0,0 +1,38 @@
+import { LitElement, html } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+import { styles } from '../styles/chat-stage.js';
+import './chat-avatar.js';
+export interface ChatStage {
+  pagetitle: string;
+  svgIcon: string;
+  isEnabled: boolean;
+}
+
+@customElement('chat-stage')
```

Comment:
i'm not sure this needs another component. It will cause you to recopy the styles.

Reply:
But it may get extended by customers. I think it doesn't hurt to keep it encapsulated.